boards/arm/arduino_zero/doc/arduino_zero.rst

```diff
@@ -82,6 +84,14 @@ SPI Port
 The SAMD21 MCU has 6 SERCOM based SPIs.  On the Arduino Zero, SPI4 is
 available on the 6 pin connector at the edge of the board.
 
+USB Device Port
+===============
+
+The SAMD21 MCU has a USB device port that can be used to communicate
+with a host PC.  See the :ref:`subsys/usb` sample applications for
```

Comment:
The :ref: tag takes the label name of the target location you want to link to, for example, the label that's at the top of the sample app README.rst files, so update these :ref: uses to be:

    See the :ref:`usb-samples` sample applications ...
    ... such as the :ref:`usb_cdc-acm` sample, which sets up ...

Reply:
Done, thanks for the pointers.